js/utils/musicutils.js

```diff
@@ -2883,17 +2898,17 @@ function getNote(
         } else if (noteArg in STOSHARP) {
             noteArg = STOSHARP[noteArg];
         }
-
+        
```

Comment:
no need for spaces here

Reply:
@walterbender  I have checked in the master branch's code, in that also there is a space.
And I think the space is needed, because there one if block is ending and the next one is starting.  If no space then it can cause confusion.